build/configs/common_download.sh

```diff
@@ -43,7 +43,15 @@ if [ -z ${DEFAULT_PORT} ]; then
 			DEFAULT_PORT=$device
 		fi
 	done
+	if [ -n ${DEFAULT_PORT} ]; then
+		if [ -n ${CONFIG_ARCH_BOARD_RTL8730E} ]; then
+			highest_number=$(echo "$DEFAULT_PORT" | grep -o '[0-9]\+')
+			desired_number=$((highest_number - 2))
+			DEFAULT_PORT="/dev/ttyUSB${desired_number}"
```

Comment:
Will this work for all versions of the 8730 board ?? 
If we know that 8730 needs ttyUSB1, then why do we need to do this arithematic ? We can just hardcode the required port number.

Reply:
yes we can, but if it opens 3 4 5 6 , then we need to download using ttyUSB4 right?